odoo/addons/base/models/res_config.py

```diff
@@ -628,6 +632,8 @@ def execute(self):
         lm = len('module_')
         for name, module in classified['module']:
             if int(self[name]):
+                if module and module.state == "installed":
```

Comment:
Since `module` is a possibly empty record, this can just use `module.state == 'installed`.